### PR TITLE
Handle sandbox realms in getRealms command.

### PR DESCRIPTION
**Pull request by jgraham**
We want to return all the Window realms with the name of the sandbox, if any.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/270.html" title="Last updated on Oct 15, 2022, 2:11 PM UTC (d5e49a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/270/582eed0...d5e49a3.html" title="Last updated on Oct 15, 2022, 2:11 PM UTC (d5e49a3)">Diff</a>

### CODE REVIEW

**Comment by juliandescottes** [comment]
@sadym-chromium Hi Maksim! For information this came up [during our implementation of getRealms](https://phabricator.services.mozilla.com/D156049), and for now we are moving forward based on the proposal in this PR. If you have any early feedback about this change, we're eager to know :)

**Comment by sadym-chromium** [comment]
We faced the same question. The question is what to do with the sandboxes created on the initial `about:blank` page. It looks like the specification implies the sandboxes created there should be transferred to the new page because of the ["that both the initial `about:blank` Document, and the new Document that is about to be created, will share the same Window object"](https://html.spec.whatwg.org/multipage/browsers.html#navigating-across-documents:is-initial-about:blank).

**Review by sadym-chromium — APPROVED** [review]
LGTM

**Comment by whimboo** [comment]
@jgraham do you also want to have a look at these proposed changes?

Btw the PR needs an update given that it has conflicts.

File: index.bs [hunk 1]
```diff
@@ -3828,7 +3844,8 @@ BaseRealmInfo = {
 WindowRealmInfo = {
   BaseRealmInfo,
   type: "window",
-  context: BrowsingContext
+  context: BrowsingContext,
+  ?sandbox: Text
```
Comment (whimboo) [inline index.bs:3848]: There is a CDDL failure.
```suggestion
  ?sandbox: text
```

**Pull request merged**